docs/user/guides/iatp-mock.md

```diff
@@ -0,0 +1,61 @@
+# IATP Mock Setup
```

Comment:
Perhaps remarking on the fact that this can be used, but is no longer necessary?

Reply:
added Optional ;)